EIPS/eip-1.md

```diff
@@ -56,13 +56,13 @@ Once the idea has been vetted, your next responsibility will be to present (by m
 
 For Core EIPs, given that they require client implementations to be considered **Final** (see "EIPs Process" below), you will need to either provide an implementation for clients or convince clients to implement your EIP. 
 
-The best way to get client implementers to review your EIP is to present it on an AllCoreDevs call. You can request to do so by posting a comment linking your EIP on an [AllCoreDevs agenda GitHub Issue].  
+The best way to get client implementers to review your EIP is to present it on an AllCoreDevs call. You can request to do so by posting a comment linking your EIP on an [AllCoreDevs agenda GitHub Issue](https://github.com/ethereum/pm/issues).  
```

Comment:
I very much prefer inline links like this, but we should remove them from the Bibliography section below as we migrate to this style.  Ideally, we would get rid of all non-inline links at once and just delete that whole section, but I'll accept iterative improvements.  😊